src/helpers.js

```diff
@@ -1,3 +1,6 @@
+import { isFunction } from "lodash"
```

Comment:
Wouldn't be too bad of a dependency if it just used the [sub-module dependency](https://www.npmjs.com/package/lodash.isfunction).

But pulling all of lodash in for this should be a no-go.